content/ch-applications/vqe-molecules.ipynb

```diff
@@ -361,7 +361,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Assume the depth setting is set to $d$. Then, RyRz has $n\\times (d+1)\\times 2$ parameters, Ry with linear entanglement has $2n\\times(d + \\frac{1}{2})$ parameters, and Ry with full entanglement has $d\\times n\\times \\frac{(n + 1)}{2} + n$ parameters."
+    "Assume the depth setting is set to $d$. Then, RyRz has $n\\times (d+1)\\times 2$ parameters, Ry with linear entanglement has $n\\times (d+1)$ parameters, and Ry with full entanglement has $n\\times (d+1)$ parameters."
```

Comment:
Not sure if this is intentional, but if this is a suggested change, can you make it in another PR please?

Reply:
Actually I am trying to do that same but don't know how XD

Reply:
I have again changed that to what it was before. The best I can do now